lib/phlex/context.rb

```diff
@@ -76,7 +76,17 @@ def _attributes(attributes)
 
       buffer = first_render ? buffer = +"" : buffer = @_target
 
+      attributes.keys.each do |key|
+        if key.match /[<>&"']/
+          raise ArgumentError, <<~MESSAGE
```

Comment:
Rather than correct unsafe attribute names, I decided to raise because checking for their presence is much faster than gsubbing, even if they’re not there. I think this is a good approach anyway as you should never expect to have unsafe attribute names and it's better to be noisy about it.